tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/openshift/api/tools
 
-go 1.23.0
+go 1.24.2
+
+toolchain go1.24.3
```

Comment:
We currently don't have 1.24 images, do we?

Reply:
Doesn't look like it :).

Do we have an ETA on when we would have them? I'm assuming we will need to have them before the next kube rebase since kube-api-linter should be following the same Go version as upstream kube